catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -144,7 +171,7 @@
  * services.
  */
 public abstract class AbstractCswSource extends MaskableImpl
-        implements FederatedSource, ConnectedSource, ConfiguredService {
+        implements CatalogStore, FederatedSource, ConnectedSource, ConfiguredService {
```

Comment:
I still don't believe CatalogStore belongs at this level - this capability is only used for the Registry and nothing else.